recipe/meta.yaml

```diff
@@ -31,14 +31,11 @@ requirements:
 
 test:
   files:
-    - ice_skating.py
-    - male1.txt
-    - import_test.py
-    - test.sh
+    - tests/
   commands:
-    - bash test.sh  # [linux]
-    - python ice_skating.py  # [osx or win]
-    - python import_test.py  # [osx or win]
+    #- bash tests/test.sh  # [linux]
```

Comment:
Ok. I'm not sure how to make it exit the Python interpreter. I copied this method from the mayavi feedstock. Theirs somehow works. I don't quite know how that script functions. Odd that it passed in staged recipes and not here.